### PR TITLE
sap_hypervisor_node_preconfigure: Fix role name

### PR DESCRIPTION
**Pull request by newkit**
Added FQRN for role used in example playbook for sap_hypervisor_node_preconfigure.

### CODE REVIEW

**Comment by kksat** [comment]
@newkit I suppose you by mistake want to merge into main

File: playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml [hunk 1]
```diff
@@ -4,8 +4,5 @@
   serial: 1
   vars:
     sap_hypervisor_node_platform: redhat_ocp_virt
-
-  tasks:
-    - name: Include Role
-      ansible.builtin.include_role:
-        name: sap_hypervisor_node_preconfigure
+  roles:
```
Comment (kksat) [inline playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml:7]: this is not correct syntax 

see https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html#using-roles-at-the-play-level

**Review by kksat — COMMENTED** [review]
see comments

File: playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml (hunk 1 above)
Comment (newkit) [reply playbooks/sample-sap-hypervisor-redhat_ocp_virt-preconfigure.yml:7]: Well, thats actually what this PR should address. And yes, it was accidentally (auto suggested) created agains main, closing.

**Comment by newkit** [comment]
This should not have been created against main branch.

**Pull request closed without merging**